vgg16_interpretation.ipynb

```diff
@@ -131,7 +140,7 @@
       },
       "source": [
         "### Step 4 Use activation maximization to visualize the images that maximize output of each filter in the prediction layer. Prediction layer is the layer in the model with 1000 filters, each representing a separate class of dataset.\n",
-        "### Question: What features has the model learnt for class *Volcano* and *Baseball*, separately? How about other classes? Share with your table the interesting visualizations you get!"
+        "### Question: What features has the model learnt for class *Volcano* and *Baseball*, separately? How about other classes? Share with your table the interesting visualizations you get! Modify variable *filter_idx* to explore the features learned by different filters in the prediction layer."
```

Comment:
One suggestion we discussed was to again refer to the description gist in Step 2 if they didnt know what index is . Vlcano and Baseball